test/dosemu_tap_interface.sh

```diff
@@ -12,7 +12,7 @@
 #      chmod 0440 /etc/sudoers.d/dosemu
 
 if [ "$1" = "setup" ] ; then
-	ip tuntap add mode tap user ${SUDO_UID} tap0
+	ip tuntap add mode tap user ${SUDO_UID} group dosemu2 tap0
```

Comment:
Ingenious. :)

Reply:
Actually it's a bit of a guess, the docs for ip tuntap seem very sparse. It takes and displays both user and group values. Currently I'm assuming that it allows both user and group access, but I don't know for sure.
~~~
root@calypso:~# ip tuntap list
tap0: tap persist user 500 group 998
~~~

Reply:
I think its a good guess.
We'll see when the packaged test
comes next week.
Although with such unlucky dj64dev
events, it will certainly find its way to
the failure.

Reply:
https://github.com/dosemu2/fdpp/actions/runs/11855896092/job/33041081809
As ingenious as it is, it
didn't work. :(
Any other work-around?